src/components/articles/reportArticleForm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/articleStyles/reportArticleForm.scss';
+
+const InputFields = (props) => {
+
+	const { value, id, handleChange, issue } = props;
```

Comment:
change line 5  from
```js
const InputFields = (props) => {
```
to 
```js
const InputFields = ({ value, id, handleChange, issue }) => {
```
delete line 7

Reply:
The change has been made